core/node/node_framework/examples/main_node.rs

```diff
@@ -46,17 +46,17 @@ use zksync_node_framework::{
             tx_sink::TxSinkLayer,
         },
     },
-    service::ZkStackService,
+    service::{ZkStackService, ZkStackServiceBuilder, ZkStackServiceError},
 };
 
 struct MainNodeBuilder {
-    node: ZkStackService,
+    node: ZkStackServiceBuilder,
```

Comment:
nit: probably `node_builder`?

Reply:
No opinion here. IMHO, the code is self-descriptive enough to understand that we're building a node so extra verbosity is not mandatory, but can change if you're feeling strongly about that.

Reply:
>if you're feeling strongly about that.

not at all